demo/locales/en.json

```diff
@@ -45,7 +45,9 @@
   "DEMO_CUSTOM_INTRO_TWO": "Press the button below to add a custom asset.",
   "DEMO_DASH": "DASH",
   "DEMO_DASH_IF": "DASH-IF",
+  "DEMO_DEFAULT_AUDIO_CODEC": "Default audio codec",
```

Comment:
To be consistent with the other configuration value names, these should be capitalized. "Default Audio Codec" and "Default Video Codec".

Reply:
Done!